docs/source/distributed.tensor.parallel.rst

```diff
@@ -32,7 +32,7 @@ Tensor Parallelism supports the following parallel styles:
 To simply configure the nn.Module's inputs and outputs with DTensor layouts
 and perform necessary layout redistributions, without distribute the module
 parameters to DTensors, the following classes can be used in
-the ``parallelize_plan``of ``parallelize_module``:
+the `parallelize_plan` of `parallelize_module`:
```

Comment:
I thought that we need double backtick for code type and that single backtick is italics, but I may be wrong.

Reply:
It's kind of weird to me too, but the lastest main docs looks wrong format so I switched back to single backtick, see lines before this API https://pytorch.org/docs/main/distributed.tensor.parallel.html#torch.distributed.tensor.parallel.PrepareModuleInput

I'll play around with the code types a bit more and see the refreshed docs before landing

Reply:
Maybe it is missing the space between the last backtick of \`\`parallelize_plan\`\` and of, causing it to render improperly?

Reply:
good point, lm try that instead

Reply:
adjusted multiple places to double backtick , but the built doc still rendering the old version, gonna land this first and see the doc